behat-tests/features/acquia_lift_element_variations.feature

```diff
@@ -1,170 +1,198 @@
-#Assumes Marketer role.
+@api @javascript @insulated @campaign
 Feature: Personalize elements variations can be edited for an existing campaign.
   In order to manage element variations in context
   As a site marketer
   I want the ability to add, edit, and delete existing personalize element variations in context.
 
-  @api @javascript @campaign
   Scenario: Add personalize elements to a campaign
+    # I have a campaign.
+    # I login with the marketer role.
+    # I am on an article page.
     Given "acquia_lift" agents:
-      | machine_name                    | label |
+      | machine_name                    | label                           |
       | testing-campaign-add-variations | Testing campaign add variations |
-    And I am logged in as a user with the "Marketer" role
+    And I am logged in as a user with the "access administration pages,access toolbar,administer visitor actions,manage personalized content" permission
     And I am on the homepage
```

Comment:
"logged in as a user with the permission" step would land the user on the home page, then the next "And I am viewing an 'Article' content" (from Behat Drupal Extension) creates the article node and then sends the user to the node page (I think).

To be serious, I am not a fan of how the language as defined upstream. I feel a better line would be "And I go to an article of:", at least. This would read much more clear of what's going on, which you would not have been confused by it then.

I in fact think we can enhance quite a bit of things as defined in Behat Drupal Extension haha, but that's a much larger topic, and is out of the scope for now.